.env.test

```diff
@@ -1,2 +1,3 @@
 LANGUAGE_PREFERENCE_COOKIE_NAME='openedx-language-preference'
 LMS_BASE_URL='http://localhost:18000'
+EXAMS_BASE_URL='http://exams'
```

Comment:
Question: is there a reason this can be 'http://exams' and not 'http://localhost:18740', or is this moot since it's just for testing and doesn't really matter since it's being mocked anyway?